src/main/webapp/ts/profile.ts

```diff
@@ -1,8 +0,0 @@
-async function getProfile() {
```

Comment:
Isn't profile.js referenced in logged_in_homepage.html? If you did mean to erase this (e.g., because it's not needed anymore), consider removing the reference.

Reply:
I deleted the reference, I think Sam made a different import in his latest PR